packages/cli/README.md

```diff
@@ -41,17 +41,27 @@ versions are incompatible with the Android command line tools.
 ### Get the Android command line tools
 Download a version of Android command line tools that is compatible with your OS from
 [https://developer.android.com/studio#command-tools](https://developer.android.com/studio#command-tools).
-Create a folder and extract the downloaded file into it.
+Create a folder and extract the downloaded file into it. This will further install the androidSdk and android SDK manager without needing to install the whole Android IDE.
 
 ### Tell Bubblewrap where the JDK and Android command line tools are
 When running `bubblewrap` for the first time, it will ask where it can find the JDK and Android command
 line tools. So, take note of the location where both were decompressed.
+- The **OpenJDK** path should contain `bin,com,include ..`
```

Comment:
Those two examples are a bit disconnected. Can we add something like:
"To ensure if you are taking note of the correct location, check if each directory contains the following files:"